feature/interface/ip/ipv6_link_local/otg_tests/ipv6_link_local/README.md

```diff
@@ -0,0 +1,39 @@
+# RT-5.8: IPv6 Link Local
+
+## Summary
+
+Configure an IPv6 address which is in link local scope. Verify the link local IPv6 address exists by checking the state path.
+
+## Procedure
+
+* Sub Test #1 - Configure IPv6 link local
+  * Configure DUT port 1 and OTG port 1 with an IPv6 link local scope IP address
+  * Validate config and state paths are set
+
```

Comment:
Curious, should you have a 3rd subtest to configure Global unicast address as well and then unconfigure it and see if the originally configured linklocal address continues to stay, serves traffic?

Similarly, disable/enable the port and see if the configured link-local address stays?